openreview/journal/webfield/editorsInChiefWebfield.js

```diff
@@ -300,6 +302,34 @@ var formatData = function(
     };
   });
 
+  archivedActionEditors.members.forEach(function(actionEditor, index) {
+    actionEditorStatusById[actionEditor.id] = {
```

Comment:
If this is the same function that's used for actionEditors on line 278, I think it would be better to combine them and do

```js
const allActionEditors = actionEditors.members.concat(archivedActionEditors.members)
allActionEditors.forEach(function(actionEditor, index) {
  ...
```